hs_core/migrations/0051_auto_20201021_2357.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-10-21 23:57
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('contenttypes', '0002_remove_content_type_name'),
+        ('hs_core', '0050_auto_20200611_1912'),
+    ]
+
```

Comment:
I had conflicts with migrations. I may need to start over and redo the migration clean.